pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 testpaths = "convert2rhel/unit_tests"
-addopts = --cov-fail-under=83 --tb=native
+addopts = --cov-fail-under=71 --tb=native
```

Comment:
previously coverage setup was incorrect, with added coverage we have to lower our acceptance limit

Reply:
Lets put the coverage changes in a separate PR

Reply:
@SpyTec Ok, I'll cherry pick this commit into another MR